response_operations_ui/controllers/collection_exercise_controllers.py

```diff
@@ -108,6 +120,27 @@ def update_event(short_name, period, tag, timestamp):
     return True
 
 
+def create_collection_exercise_event(collection_exercise_id, tag, timestamp):
```

Comment:
`create_collection_exercise_event_date`?

Reply:
In the collection exercise service these are just events for the collection exercise that have a date. So this controller is creating the event in the collection exercise service.